service/mapper/transitions_test.go

```diff
@@ -0,0 +1,1233 @@
+// Copyright 2021 Optakt Labs OÜ
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+package mapper
+
+import (
+	"io"
+	"sync"
+	"testing"
+
+	"github.com/rs/zerolog"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/onflow/flow-go/ledger"
+	"github.com/onflow/flow-go/ledger/complete/mtrie/trie"
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/optakt/flow-dps/models/dps"
+	"github.com/optakt/flow-dps/testing/mocks"
+)
+
+// For path/payload/trie/trie update test variables, see testing globals defined in pathings_test.go.
+var testLog = zerolog.New(io.Discard)
+
+func TestNewTransitions(t *testing.T) {
+	t.Run("nominal case, without options", func(t *testing.T) {
+		load := &mocks.Loader{}
+		chain := &mocks.Chain{}
+		feed := &mocks.Feeder{}
+		index := &mocks.Writer{}
+
+		tr := NewTransitions(testLog, load, chain, feed, index)
+
+		assert.NotNil(t, tr)
+		assert.Equal(t, load, tr.load)
+		assert.Equal(t, chain, tr.chain)
+		assert.Equal(t, feed, tr.feed)
+		assert.Equal(t, index, tr.index)
+		assert.NotNil(t, tr.once)
+		assert.Zero(t, tr.cfg)
```

Comment:
We shouldn't assume the default configuration to correspond to the zero value. Instead, you can use `assert.Equal(t, tr.cfg, DefaultConfig)`.